test/jest/reportingLogger.spec.ts

```diff
@@ -0,0 +1,147 @@
+import { IRateLimiter, RateLimiter, ReportingLogger } from '../../src/logging/reportingLogger';
+import { LogRequestSeverity } from '../../src/logging/logRequest';
+import { ErrorCodes } from '../../src/logging/errorCodes';
+
+describe('ReportingLogger', () => {
+    let mpInstance: any;
+    let logger: ReportingLogger;
+    const sdkVersion = '1.2.3';
+    let mockFetch: jest.Mock;
+    const accountId = '1234567890';
+    beforeEach(() => {
+        mockFetch = jest.fn().mockResolvedValue({ ok: true });
+        global.fetch = mockFetch;
+        
+        mpInstance = {
+            _Helpers: {
```

Comment:
This may be future work but if it's easy enough, we should rewrite `createServiceUrl` so that it isn't as tightly coupled to the full SDK instance or Store, or make it a function within `Store`.

If it's future work, we should collect some of these refactors into a launch pad.